events-to-histogram/src/main.rs

```diff
@@ -81,7 +81,7 @@ async fn main() -> Result<()> {
     loop {
         match consumer.recv().await {
             Ok(m) => {
-                log::debug!(
+                tracing::debug!(
```

Comment:
For the sake of clarity, could the various macros be imported rather than using the fully qualified name every time.

See https://github.com/STFC-ICD-Research-and-Design/supermusr-data-pipeline/blob/main/digitiser-aggregator/src/main.rs#L18